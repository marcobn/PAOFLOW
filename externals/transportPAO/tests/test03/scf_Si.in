 &control
    calculation = 'scf'
    restart_mode='from_scratch',
    prefix='silicon',
    tprnfor = .true.
    pseudo_dir = '../pseudo/'
    outdir = './SCRATCH/'
 /
 &system    
    ibrav=  2, celldm(1) =10.599478, nat=  2, ntyp= 1,
    ecutwfc =30.0,  nbnd = 16
    nr1 = 32
    nr2 = 32
    nr3 = 32
 /
 &electrons
    mixing_mode = 'plain'
    mixing_beta = 0.7 
    conv_thr =  1.0d-8
 /
ATOMIC_SPECIES
 Si  28.086  Si.pw-mt_fhi.UPF
ATOMIC_POSITIONS (alat)
 Si 0.00 0.00 0.00 
 Si 0.25 0.25 0.25 
K_POINTS {automatic}
  8 8 8 1 1 1
