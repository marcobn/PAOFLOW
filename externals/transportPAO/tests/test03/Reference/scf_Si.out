
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:40:46 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/scf_Si.in
               file Si.pw-mt_fhi.UPF: wavefunction(s)  4f renormalized

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


   Info: using nr1, nr2, nr3 values from input
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          77      77     25                 1102     1102     206
     Max          78      78     26                 1103     1103     209
     Sum         463     463    151                 6615     6615    1243
 


     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.5995  a.u.
     unit-cell volume          =     297.7100 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=           16
     kinetic-energy cutoff     =      30.0000  Ry
     charge density cutoff     =     120.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = SLA-PW ( 1 4 0 0 0)
     EXX-fraction              =        0.00

     celldm(1)=  10.599478  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Si read from file:
     ../pseudo/Si.pw-mt_fhi.UPF
     MD5 check sum: d581ac71374b31d983c532c8bb0e3c43
     Pseudo is Norm-conserving, Zval =  4.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.1
     Using radial grid of  495 points,  3 beta functions with: 
                l(1) =   0
                l(2) =   1
                l(3) =   3

     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08600     Si( 1.00)

     48 Sym. Ops., with inversion, found (24 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Si  tau(   2) = (   0.2500000   0.2500000   0.2500000  )

     number of k points=    60
                       cart. coord. in units 2pi/alat
        k(    1) = (  -0.0625000   0.0625000   0.0625000), wk =   0.0078125
        k(    2) = (  -0.1875000   0.1875000  -0.0625000), wk =   0.0234375
        k(    3) = (  -0.3125000   0.3125000  -0.1875000), wk =   0.0234375
        k(    4) = (  -0.4375000   0.4375000  -0.3125000), wk =   0.0234375
        k(    5) = (   0.4375000  -0.4375000   0.5625000), wk =   0.0234375
        k(    6) = (   0.3125000  -0.3125000   0.4375000), wk =   0.0234375
        k(    7) = (   0.1875000  -0.1875000   0.3125000), wk =   0.0234375
        k(    8) = (   0.0625000  -0.0625000   0.1875000), wk =   0.0234375
        k(    9) = (  -0.0625000   0.3125000   0.0625000), wk =   0.0234375
        k(   10) = (  -0.1875000   0.4375000  -0.0625000), wk =   0.0468750
        k(   11) = (  -0.3125000   0.5625000  -0.1875000), wk =   0.0468750
        k(   12) = (   0.5625000  -0.3125000   0.6875000), wk =   0.0468750
        k(   13) = (   0.4375000  -0.1875000   0.5625000), wk =   0.0468750
        k(   14) = (   0.3125000  -0.0625000   0.4375000), wk =   0.0468750
        k(   15) = (   0.1875000   0.0625000   0.3125000), wk =   0.0468750
        k(   16) = (  -0.0625000   0.5625000   0.0625000), wk =   0.0234375
        k(   17) = (  -0.1875000   0.6875000  -0.0625000), wk =   0.0468750
        k(   18) = (   0.6875000  -0.1875000   0.8125000), wk =   0.0468750
        k(   19) = (   0.5625000  -0.0625000   0.6875000), wk =   0.0468750
        k(   20) = (   0.4375000   0.0625000   0.5625000), wk =   0.0468750
        k(   21) = (   0.3125000   0.1875000   0.4375000), wk =   0.0468750
        k(   22) = (  -0.0625000   0.8125000   0.0625000), wk =   0.0234375
        k(   23) = (   0.8125000  -0.0625000   0.9375000), wk =   0.0468750
        k(   24) = (   0.6875000   0.0625000   0.8125000), wk =   0.0468750
        k(   25) = (   0.5625000   0.1875000   0.6875000), wk =   0.0468750
        k(   26) = (   0.4375000   0.3125000   0.5625000), wk =   0.0468750
        k(   27) = (  -0.0625000  -0.9375000   0.0625000), wk =   0.0234375
        k(   28) = (  -0.1875000  -0.8125000  -0.0625000), wk =   0.0468750
        k(   29) = (  -0.3125000  -0.6875000  -0.1875000), wk =   0.0468750
        k(   30) = (  -0.0625000  -0.6875000   0.0625000), wk =   0.0234375
        k(   31) = (  -0.1875000  -0.5625000  -0.0625000), wk =   0.0468750
        k(   32) = (  -0.0625000  -0.4375000   0.0625000), wk =   0.0234375
        k(   33) = (  -0.1875000   0.1875000   0.1875000), wk =   0.0078125
        k(   34) = (  -0.3125000   0.3125000   0.0625000), wk =   0.0234375
        k(   35) = (  -0.4375000   0.4375000  -0.0625000), wk =   0.0234375
        k(   36) = (   0.4375000  -0.4375000   0.8125000), wk =   0.0234375
        k(   37) = (   0.3125000  -0.3125000   0.6875000), wk =   0.0234375
        k(   38) = (   0.1875000  -0.1875000   0.5625000), wk =   0.0234375
        k(   39) = (  -0.1875000   0.4375000   0.1875000), wk =   0.0234375
        k(   40) = (  -0.3125000   0.5625000   0.0625000), wk =   0.0468750
        k(   41) = (   0.5625000  -0.3125000   0.9375000), wk =   0.0468750
        k(   42) = (   0.4375000  -0.1875000   0.8125000), wk =   0.0468750
        k(   43) = (   0.3125000  -0.0625000   0.6875000), wk =   0.0468750
        k(   44) = (  -0.1875000   0.6875000   0.1875000), wk =   0.0234375
        k(   45) = (   0.6875000  -0.1875000   1.0625000), wk =   0.0468750
        k(   46) = (   0.5625000  -0.0625000   0.9375000), wk =   0.0468750
        k(   47) = (   0.4375000   0.0625000   0.8125000), wk =   0.0468750
        k(   48) = (  -0.1875000  -1.0625000   0.1875000), wk =   0.0234375
        k(   49) = (  -0.3125000  -0.9375000   0.0625000), wk =   0.0468750
        k(   50) = (  -0.1875000  -0.8125000   0.1875000), wk =   0.0234375
        k(   51) = (  -0.3125000   0.3125000   0.3125000), wk =   0.0078125
        k(   52) = (  -0.4375000   0.4375000   0.1875000), wk =   0.0234375
        k(   53) = (   0.4375000  -0.4375000   1.0625000), wk =   0.0234375
        k(   54) = (   0.3125000  -0.3125000   0.9375000), wk =   0.0234375
        k(   55) = (  -0.3125000   0.5625000   0.3125000), wk =   0.0234375
        k(   56) = (   0.5625000  -0.3125000   1.1875000), wk =   0.0468750
        k(   57) = (   0.4375000  -0.1875000   1.0625000), wk =   0.0468750
        k(   58) = (  -0.3125000  -1.1875000   0.3125000), wk =   0.0234375
        k(   59) = (  -0.4375000   0.4375000   0.4375000), wk =   0.0078125
        k(   60) = (   0.4375000  -0.4375000   1.3125000), wk =   0.0234375

     Dense  grid:     6615 G-vectors     FFT dimensions: (  32,  32,  32)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.04 Mb     (    144,   16)
        NL pseudopotentials             0.05 Mb     (    144,   22)
        Each V/rho on FFT grid          0.09 Mb     (   6144)
        Each G-vector array             0.01 Mb     (   1102)
        G-vector shells                 0.00 Mb     (    115)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         0.14 Mb     (    144,   64)
        Each subspace H/S matrix        0.06 Mb     (  64,  64)
        Each <psi_i|beta_j> matrix      0.01 Mb     (     22,   16)
        Arrays for rho mixing           0.75 Mb     (   6144,   8)

     Initial potential from superposition of free atoms

     starting charge    7.99872, renormalised to    8.00000
     Starting wfc are   32 randomized atomic wfcs

     total cpu time spent up to now is        2.0 secs

     per-process dynamical memory:     4.5 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    30.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.1

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  6.40E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        4.3 secs

     total energy              =     -15.86378650 Ry
     Harris-Foulkes estimate   =     -15.87895305 Ry
     estimated scf accuracy    <       0.05104631 Ry

     iteration #  2     ecut=    30.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  6.38E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        5.1 secs

     total energy              =     -15.86608298 Ry
     Harris-Foulkes estimate   =     -15.86634228 Ry
     estimated scf accuracy    <       0.00182287 Ry

     iteration #  3     ecut=    30.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.28E-05,  avg # of iterations =  3.0

     total cpu time spent up to now is        6.1 secs

     total energy              =     -15.86610402 Ry
     Harris-Foulkes estimate   =     -15.86610940 Ry
     estimated scf accuracy    <       0.00004515 Ry

     iteration #  4     ecut=    30.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  5.64E-07,  avg # of iterations =  3.0

     total cpu time spent up to now is        7.1 secs

     total energy              =     -15.86610527 Ry
     Harris-Foulkes estimate   =     -15.86610526 Ry
     estimated scf accuracy    <       0.00000001 Ry

     iteration #  5     ecut=    30.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.40E-10,  avg # of iterations =  3.0

     total cpu time spent up to now is        8.2 secs

     End of self-consistent calculation

          k =-0.0625 0.0625 0.0625 (   863 PWs)   bands (ev):

    -6.2208   4.4907   4.9769   4.9769   7.2805   7.6660   7.6660   7.8866
    12.5179  12.5989  12.5989  15.6579  16.1742  16.1742  19.8306  26.0458

          k =-0.1875 0.1875-0.0625 (   839 PWs)   bands (ev):

    -5.9712   3.0100   4.2337   4.8623   7.3321   7.8971   8.1034   9.3931
    11.8092  12.5099  13.4376  15.4588  16.7561  17.8421  21.2560  23.2612

          k =-0.3125 0.3125-0.1875 (   830 PWs)   bands (ev):

    -5.3707   0.9263   3.7631   4.4325   6.8805   8.3855   8.6394  11.0450
    11.8072  13.1209  13.7469  16.4672  18.6647  19.8700  20.5719  21.7145

          k =-0.4375 0.4375-0.3125 (   825 PWs)   bands (ev):

    -4.5426  -0.8230   3.5489   4.0538   6.5705   8.4965   8.5032  12.4196
    13.1460  13.6778  14.9790  16.6142  18.1377  18.3503  20.7167  22.5741

          k = 0.4375-0.4375 0.5625 (   826 PWs)   bands (ev):

    -4.1410  -1.4607   3.5870   3.8537   6.4955   8.3216   8.5793  12.5585
    14.0433  15.2892  15.5859  15.8773  16.1788  17.3376  22.6297  23.9382

          k = 0.3125-0.3125 0.4375 (   836 PWs)   bands (ev):

    -4.8246  -0.3087   3.8525   3.8788   6.6654   8.2983   8.7916  11.6539
    13.2662  13.7565  13.7740  17.5526  17.9407  19.1743  20.6011  21.7275

          k = 0.1875-0.1875 0.3125 (   828 PWs)   bands (ev):

    -5.6055   1.6081   4.1416   4.2904   7.0432   8.0792   8.7843  10.3647
    11.8960  12.6761  13.8193  15.8315  18.2396  19.0945  21.7515  21.7783

          k = 0.0625-0.0625 0.1875 (   842 PWs)   bands (ev):

    -6.0950   3.7258   4.5906   4.7124   7.3604   7.6644   8.1938   8.6550
    11.8879  12.7644  13.0309  15.6698  16.3853  16.9178  20.6714  24.7444

          k =-0.0625 0.3125 0.0625 (   840 PWs)   bands (ev):

    -5.8455   2.8273   4.0777   4.1267   6.9745   8.3314   8.9842   9.3023
    11.3405  13.0936  13.8987  16.0624  16.7991  17.6340  22.1310  23.1133

          k =-0.1875 0.4375-0.0625 (   836 PWs)   bands (ev):

    -5.3582   1.4109   3.1665   3.8914   7.1198   8.4481   9.4369   9.7264
    12.1871  13.6594  14.9033  16.4308  17.8025  19.4586  20.3013  21.9025

          k =-0.3125 0.5625-0.1875 (   822 PWs)   bands (ev):

    -4.5780  -0.2952   2.6685   3.5222   7.2468   8.5674   9.7521  10.0936
    13.7266  14.5048  15.4832  16.6145  17.9759  19.4120  21.0098  21.9276

          k = 0.5625-0.3125 0.6875 (   826 PWs)   bands (ev):

    -3.8901  -1.3949   2.3878   3.4403   7.2602   8.3785   9.6487  11.9120
    13.1850  14.8003  15.8530  16.2347  17.7961  19.2299  22.3656  23.1193

          k = 0.4375-0.1875 0.5625 (   822 PWs)   bands (ev):

    -4.3014  -0.7698   2.3339   3.6983   7.4618   8.3521   9.5755  11.2261
    13.1432  13.9263  15.8351  16.8386  18.3753  19.6678  20.6062  22.1094

          k = 0.3125-0.0625 0.4375 (   831 PWs)   bands (ev):

    -5.1289   0.8135   2.6485   4.1612   7.7739   8.1817   9.4355   9.9247
    11.7979  14.4484  14.9238  16.5213  18.2111  19.2841  20.5507  22.1298

          k = 0.1875 0.0625 0.3125 (   832 PWs)   bands (ev):

    -5.7243   2.2172   3.6973   4.5011   7.4471   7.9078   8.8050   9.8257
    11.5433  12.9014  14.1113  15.7766  17.2062  18.7721  21.6810  22.3151

          k =-0.0625 0.5625 0.0625 (   831 PWs)   bands (ev):

    -4.9852   0.8888   3.0416   3.1582   6.2091   7.7872  10.6593  10.8466
    12.9187  14.2241  16.0419  17.5862  17.9080  18.9815  20.4376  20.7101

          k =-0.1875 0.6875-0.0625 (   826 PWs)   bands (ev):

    -4.2887  -0.2643   2.3132   2.8380   6.5090   7.6457  10.6174  11.7542
    14.3188  15.7178  15.7571  17.3933  18.5350  18.7562  20.3689  21.4195

          k = 0.6875-0.1875 0.8125 (   824 PWs)   bands (ev):

    -3.4684  -1.3578   1.5314   2.8075   6.7341   9.2095  10.2150  11.6634
    13.4572  15.2449  16.4695  17.7392  18.2451  20.5740  21.8066  22.2730

          k = 0.5625-0.0625 0.6875 (   823 PWs)   bands (ev):

    -3.5378  -1.2677   1.2123   3.1229   7.0928   9.8614  10.9872  11.5006
    11.6211  13.1073  17.7382  18.5767  19.5094  20.0344  20.8692  21.4376

          k = 0.4375 0.0625 0.5625 (   824 PWs)   bands (ev):

    -4.3587  -0.3639   1.7587   3.6589   7.6168   8.8309  10.0951  10.4635
    12.2616  14.0593  16.6140  17.2598  18.2778  19.9595  20.9868  21.7550

          k = 0.3125 0.1875 0.4375 (   830 PWs)   bands (ev):

    -5.0251   0.2867   3.2457   4.0667   7.1917   8.2655   9.0912  10.6572
    12.5397  13.9825  14.2755  17.0548  18.3188  19.7638  20.4251  21.4990

          k =-0.0625 0.8125 0.0625 (   814 PWs)   bands (ev):

    -3.6802  -1.0736   2.4068   2.6039   5.8520   6.5478  12.7667  12.8470
    15.2536  15.9124  16.3837  16.6086  18.1272  19.3376  20.3309  21.9116

          k = 0.8125-0.0625 0.9375 (   815 PWs)   bands (ev):

    -2.8404  -1.9946   1.8983   2.3822   6.2658   6.9497  12.5701  13.4138
    14.3371  15.3357  16.6694  17.7820  18.2833  19.3602  20.9270  22.3332

          k = 0.6875 0.0625 0.8125 (   823 PWs)   bands (ev):

    -2.9466  -1.8294   1.2308   2.6736   6.6003   9.0560  11.1591  12.1630
    12.8643  14.2286  17.6358  18.8532  19.5559  19.8380  21.1043  21.2218

          k = 0.5625 0.1875 0.6875 (   827 PWs)   bands (ev):

    -3.6514  -1.3857   1.6895   3.2279   7.1863   9.0409  10.4904  11.6869
    12.1997  14.0819  16.4322  17.4874  19.1267  19.8407  21.3238  22.6160

          k = 0.4375 0.3125 0.5625 (   826 PWs)   bands (ev):

    -4.2121  -1.2001   3.0640   3.7630   6.9713   8.2495   8.9559  12.2158
    13.5001  14.4536  15.5541  16.3518  17.0145  18.7193  21.4828  23.2991

          k =-0.0625-0.9375 0.0625 (   815 PWs)   bands (ev):

    -2.8839  -2.0079   2.2788   2.4905   5.8602   6.2026  13.7650  13.7679
    15.2941  15.5867  16.5106  16.9674  17.5714  18.1112  21.7606  22.8650

          k =-0.1875-0.8125-0.0625 (   820 PWs)   bands (ev):

    -3.6037  -1.1343   2.0402   2.5320   6.3404   7.1928  11.5728  12.8023
    14.9129  15.4458  16.4876  17.0402  18.6159  19.5262  20.3759  21.3634

          k =-0.3125-0.6875-0.1875 (   824 PWs)   bands (ev):

    -4.0478  -0.8345   2.0567   3.0851   6.9790   9.3236   9.5208  10.7168
    14.3652  15.0601  16.1314  16.7732  17.3310  20.1282  21.8288  22.1645

          k =-0.0625-0.6875 0.0625 (   822 PWs)   bands (ev):

    -4.3844  -0.1005   2.6619   2.8284   5.9666   7.0933  11.6978  11.8387
    14.1080  15.0070  16.7543  17.6220  18.2043  19.2610  20.2202  20.9468

          k =-0.1875-0.5625-0.0625 (   832 PWs)   bands (ev):

    -4.8773   0.5869   2.6964   3.3019   6.7756   8.2365   9.7545  10.7150
    13.2121  14.6327  15.6395  17.2159  18.1688  19.3329  19.7800  21.4607

          k =-0.0625-0.4375 0.0625 (   836 PWs)   bands (ev):

    -5.4743   1.8723   3.5382   3.5822   6.5585   8.4585   9.7425   9.8878
    11.8455  13.5827  14.9585  16.7282  17.3774  18.3519  21.6027  21.7526

          k =-0.1875 0.1875 0.1875 (   833 PWs)   bands (ev):

    -5.8486   2.3419   4.5692   4.5692   6.8749   8.2428   8.2428   9.9474
    12.1031  12.1031  13.5551  15.2571  17.9882  17.9882  21.6600  23.3388

          k =-0.3125 0.3125 0.0625 (   834 PWs)   bands (ev):

    -5.4842   1.5119   3.1634   4.5766   7.5419   8.3650   8.5772  10.2900
    11.3182  13.4204  14.4737  16.0318  17.7434  19.9716  20.2070  22.3848

          k =-0.4375 0.4375-0.0625 (   824 PWs)   bands (ev):

    -4.7921   0.2106   2.1575   4.1120   7.5190   9.0783   9.1824  10.5781
    11.3987  14.7240  15.6818  16.6813  17.9720  19.7867  21.1521  21.5311

          k = 0.4375-0.4375 0.8125 (   833 PWs)   bands (ev):

    -3.9755  -1.1476   1.9856   3.6234   7.1131   9.1186   9.4939  12.3566
    12.8534  12.8932  15.8759  18.1556  18.8140  18.8957  20.6759  23.0958

          k = 0.3125-0.3125 0.6875 (   822 PWs)   bands (ev):

    -3.9599  -1.1744   2.2545   3.3322   6.8921   9.1507   9.8221  10.6657
    14.1188  15.1651  16.2418  16.4325  16.5063  20.5214  21.6565  22.8234

          k = 0.1875-0.1875 0.5625 (   831 PWs)   bands (ev):

    -4.7753   0.2071   2.8324   3.3145   6.8495   8.8254   9.5196  10.1283
    14.0188  14.4988  15.0198  17.5275  17.6918  19.2065  20.4241  21.5857

          k =-0.1875 0.4375 0.1875 (   832 PWs)   bands (ev):

    -5.2458   0.8974   3.5881   3.7029   7.1577   8.3428   9.4219   9.8231
    12.7876  13.4921  14.3412  16.7271  18.3614  19.6356  20.5521  21.0765

          k =-0.3125 0.5625 0.0625 (   831 PWs)   bands (ev):

    -4.6652   0.1298   2.2122   3.5136   7.6752   8.5294   9.2118  10.5523
    12.8219  15.0234  15.7205  16.9178  18.1290  19.5274  20.1764  22.5313

          k = 0.5625-0.3125 0.9375 (   827 PWs)   bands (ev):

    -3.8451  -0.9116   1.4867   2.9978   8.0851   8.5793   9.7563  11.5737
    12.9717  13.5415  17.3718  18.0862  18.9546  19.1150  21.1855  22.0955

          k = 0.4375-0.1875 0.8125 (   824 PWs)   bands (ev):

    -3.4026  -1.4671   1.4766   2.7140   7.6709   8.8735  10.4960  11.6276
    12.0462  15.4465  16.1858  17.4604  19.5557  20.8646  21.1971  22.0832

          k = 0.3125-0.0625 0.6875 (   826 PWs)   bands (ev):

    -4.1039  -0.5561   1.8804   2.8765   7.4235   8.5817   9.3235  11.6768
    13.9984  14.3992  16.9550  17.3158  18.5402  18.8100  20.8917  21.7877

          k =-0.1875 0.6875 0.1875 (   821 PWs)   bands (ev):

    -4.2044  -0.4910   2.1793   3.0067   6.5114   8.4269  10.4012  10.9049
    15.2007  15.5031  15.6413  16.4962  17.7424  19.5539  21.0091  22.3535

          k = 0.6875-0.1875 1.0625 (   822 PWs)   bands (ev):

    -3.4616  -1.2629   1.6647   2.3854   7.2401   8.2070  10.1623  12.7294
    13.4441  15.1884  16.7883  18.1525  18.9078  19.1671  20.9805  21.8663

          k = 0.5625-0.0625 0.9375 (   823 PWs)   bands (ev):

    -2.7217  -1.9722   1.3822   1.9256   8.3055   9.2246   9.7089  10.6943
    14.0414  15.4090  16.6556  18.0693  19.8286  21.2181  21.3439  22.1647

          k = 0.4375 0.0625 0.8125 (   824 PWs)   bands (ev):

    -3.2784  -1.4497   1.3637   2.3632   8.3983   8.7798   9.4499  11.7869
    12.8529  14.8571  17.0968  18.3678  19.2992  20.0487  20.8309  22.8063

          k =-0.1875-1.0625 0.1875 (   819 PWs)   bands (ev):

    -2.8671  -1.9430   1.4773   2.6873   6.1174   7.9031  12.5516  12.5724
    13.7426  14.2787  17.6062  18.1600  19.4339  19.4799  20.1574  21.7511

          k =-0.3125-0.9375 0.0625 (   823 PWs)   bands (ev):

    -2.7738  -1.9796   1.5638   2.1180   7.1465   8.0115  11.1104  12.1437
    14.0726  15.3618  16.6645  17.9790  19.6432  20.3747  20.9145  21.8658

          k =-0.1875-0.8125 0.1875 (   822 PWs)   bands (ev):

    -3.5509  -1.2202   1.7142   2.7949   6.2541   8.0789  11.4710  11.7455
    14.7695  15.2846  16.5988  16.8723  18.5218  20.0356  20.7289  22.4724

          k =-0.3125 0.3125 0.3125 (   827 PWs)   bands (ev):

    -5.1477   0.2764   4.1807   4.1807   6.4217   8.5106   8.5106  11.9956
    12.6595  12.6595  13.3389  17.1877  19.2193  19.2194  20.7162  21.5875

          k =-0.4375 0.4375 0.1875 (   821 PWs)   bands (ev):

    -4.7046  -0.2817   2.7916   4.0968   7.1940   8.6826   8.7688  11.5629
    12.0939  14.0488  15.4728  16.4667  18.5452  19.8857  20.2791  21.3787

          k = 0.4375-0.4375 1.0625 (   825 PWs)   bands (ev):

    -3.9741  -0.8369   1.4103   3.5652   7.0797  10.0363  10.0737  11.2849
    12.0026  12.8401  17.0701  18.3596  18.8412  20.3592  20.4509  22.2847

          k = 0.3125-0.3125 0.9375 (   820 PWs)   bands (ev):

    -3.2213  -1.5822   1.1138   3.0613   6.5410  10.2466  11.2178  11.2559
    12.3838  13.0055  18.2853  18.7783  19.3755  20.1831  20.9485  21.6265

          k =-0.3125 0.5625 0.3125 (   823 PWs)   bands (ev):

    -4.4171  -0.7960   3.0825   3.5824   7.0113   8.4029   9.2338  11.0870
    13.8250  15.0183  15.0225  16.5407  16.8823  19.7329  21.0322  22.2444

          k = 0.5625-0.3125 1.1875 (   823 PWs)   bands (ev):

    -3.8470  -1.1655   1.8893   3.1501   7.7939   8.2505  10.5113  10.8989
    13.0481  14.5263  16.1402  17.2512  18.2654  20.3951  21.0145  22.7459

          k = 0.4375-0.1875 1.0625 (   825 PWs)   bands (ev):

    -3.0920  -1.6549   1.2002   2.4945   7.5600   9.6986  10.3803  10.8397
    12.7957  14.5097  17.4012  18.5374  19.5709  20.4881  21.2259  21.9128

          k =-0.3125-1.1875 0.3125 (   827 PWs)   bands (ev):

    -3.5191  -1.4394   1.5441   3.1537   6.6724  10.1477  10.4002  10.5253
    13.4595  13.9556  17.0041  17.7328  17.8870  21.1742  22.0332  22.1791

          k =-0.4375 0.4375 0.4375 (   829 PWs)   bands (ev):

    -4.3367  -1.2440   3.9709   3.9709   6.1700   8.4206   8.4206  12.4720
    14.3945  14.3945  14.6428  16.8247  16.8247  17.1077  21.6149  24.1328

          k = 0.4375-0.4375 1.3125 (   827 PWs)   bands (ev):

    -4.0162  -1.4182   2.7935   3.7275   6.9910   8.4487   8.9552  12.7419
    13.4657  14.0145  15.2717  17.3259  17.4107  17.7796  22.2749  23.7327

     highest occupied, lowest unoccupied level (ev):     4.9769    5.8520

!    total energy              =     -15.86610535 Ry
     Harris-Foulkes estimate   =     -15.86610534 Ry
     estimated scf accuracy    <          1.2E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       3.90568556 Ry
     hartree contribution      =       1.17829767 Ry
     xc contribution           =      -4.68725582 Ry
     ewald contribution        =     -16.26283275 Ry

     convergence has been achieved in   5 iterations

     Forces acting on atoms (Ry/au):

     atom    1 type  1   force =     0.00000000    0.00000000    0.00000000
     atom    2 type  1   force =     0.00000000    0.00000000    0.00000000

     Total force =     0.000000     Total SCF correction =     0.000000

     Writing output data file silicon.save
 
     init_run     :      1.53s CPU      1.59s WALL (       1 calls)
     electrons    :      6.02s CPU      6.18s WALL (       1 calls)
     forces       :      0.04s CPU      0.03s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      1.50s CPU      1.53s WALL (       1 calls)
     potinit      :      0.01s CPU      0.03s WALL (       1 calls)

     Called by electrons:
     c_bands      :      5.18s CPU      5.30s WALL (       6 calls)
     sum_band     :      0.80s CPU      0.83s WALL (       6 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (       6 calls)
     mix_rho      :      0.01s CPU      0.00s WALL (       6 calls)

     Called by c_bands:
     init_us_2    :      0.09s CPU      0.07s WALL (     840 calls)
     cegterg      :      5.01s CPU      5.12s WALL (     360 calls)

     Called by *egterg:
     h_psi        :      4.67s CPU      5.09s WALL (    1330 calls)
     g_psi        :      0.01s CPU      0.03s WALL (     910 calls)
     cdiaghg      :      1.30s CPU      1.04s WALL (    1210 calls)

     Called by h_psi:
     add_vuspsi   :      0.06s CPU      0.08s WALL (    1330 calls)

     General routines
     calbec       :      0.10s CPU      0.12s WALL (    1390 calls)
     fft          :      0.01s CPU      0.01s WALL (      27 calls)
     fftw         :      4.80s CPU      5.22s WALL (   36400 calls)
     davcio       :      0.02s CPU      0.05s WALL (    1200 calls)
 
     Parallel routines
     fft_scatter  :      1.82s CPU      2.06s WALL (   36427 calls)
 
     PWSCF        :     8.10s CPU         8.50s WALL

 
   This run was terminated on:  20:40:54  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
